src/components/CartCheckout/CartCheckout.module.scss

```diff
@@ -0,0 +1,94 @@
+@import '@styles/variables';
+@import '@styles/breakpoints';
+
+.cart_page_title {
+  font-weight: 800;
+  font-size: 32px;
+  line-height: 41px;
+  letter-spacing: -0.01em;
+  color: $color-white;
+}
+
+.total_checkout {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 288px;
+  height: 190px;
+  padding: 24px;
+  border: 1px solid $color-icons;
+
+  @include bp-custom-up (640px) {
+    width: 592px;
+    height: 190px;
+    padding: 24px;
+  }
+
+  @include bp-x-large-up {
+    width: 368px;
+  }
+}
+
+.info {
+  width: 240px;
+
+  @include bp-custom-up (640px) {
+    width: 544px;
+  }
+  @include bp-x-large-up {
+    width: 100%;
+  }
+}
+
+.total_price {
+  font-weight: 800;
+  font-size: 32px;
+  line-height: 41px;
+  text-align: center;
+  letter-spacing: -0.01em;
+  color: $color-white;
+}
+
+.number_of_items {
+  font-weight: 600;
+  font-size: 14px;
+  line-height: 21px;
+  text-align: center;
+
+  color: $color-secondary;
+  order: 1;
+  align-self: stretch;
+  flex-grow: 0;
+  margin-bottom: 16px;
```

Comment:
move to separate variable or use calc($main-indent / 1.5). you use the value of 16px many times